src/Identity/UI/src/Microsoft.AspNetCore.Identity.UI.csproj

```diff
@@ -22,6 +22,7 @@
     <None Remove="wwwroot/**/LICENSE*" />
     <None Include="THIRD-PARTY-NOTICES.txt" Pack="true" PackagePath="/THIRD-PARTY-NOTICES.txt" />
 
+    <Content Update="wwwroot/**/*" Pack="false" />
```

Comment:
Slashes going the other way to be consistent?

Reply:
We're already mixed within the file, the `None` and `EmbeddedResource` Items above use `/` instead of `\`, I feel like if anything I should switch the cshtml slashes.